src/tasks/copy-folder-task.test.ts

```diff
@@ -15,6 +17,8 @@ import { LocalService } from '../fileServices/localService';
 import { S3Service } from '../fileServices/s3Service';
 import CopyFolderTask from './copy-folder-task';
 
+const mockHandler = {} as unknown as DatabaseTransactionHandler;
```

Comment:
Info: why is `as unknown` required here?

Reply:
It's to force the cast, since we need to define  `DatabaseTransactionHandler` but it is not used. Can be perfected with `graasp-test` later.